tests/Integration/utils/clean_up_repo.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
[Needs to be fixed]
Do we need this file in this repo? I think we can delete this file since it has served its purpose already.